package.json

```diff
@@ -2,6 +2,9 @@
   "name": "@electron/devtron",
   "version": "0.0.0-development",
   "description": "Electron DevTools Extension to track IPC events",
+  "main": "./dist/cjs/index.cjs",
+  "module": "./dist/mjs/index.mjs",
```

Comment:
Does `module` do anything here? And do we need `main` if we're already using `exports`?